docs/grid.rst

```diff
@@ -18,12 +20,13 @@ A ``Grid`` object contains data representing an electric power system. An object
 - ``dcline`` (``pandas.DataFrame``) encloses the characteristics of the HVDC lines
 - ``storage`` (``dict``) encloses information related to storage units
 
-Only the U.S. Test system presented `here <https://arxiv.org/pdf/2002.06155.pdf>`_  is
-available at this time. Thus, a ``Grid`` object can represent in addition to the full
-continental U.S., one of the three interconnections -- Eastern, Western or Texas-- or
-a combination of two interconnections.
+Two grid models representing the `U.S. <https://arxiv.org/pdf/2002.06155.pdf>`_ and
+the `European <https://arxiv.org/pdf/1806.01613.pdf>`_ power system at the transmission
+network level are available at this time. In addition to the full continental U.S.
+(ENTSO-E area), a ``Grid`` object can represent one of the the interconnection or a
```

Comment:
This looks like it is equating the U.S. and ENTSO-E?

Reply:
Changed to `In addition to the full continental U.S. or Europe`